lib/new_relic/util/http.ex

```diff
@@ -16,6 +16,14 @@ defmodule NewRelic.Util.HTTP do
   def post(url, body, headers),
     do: post(url, Jason.encode!(body), headers)
 
+  def get(url, headers) do
+    headers = Enum.map(headers, fn {k, v} -> {'#{k}', '#{v}'} end)
+
+    with {:ok, {{_, status, _}, _, body}} <- :httpc.request(:get, {'#{url}', headers}, [], []) do
```

Comment:
With no other match here, would this be fine as a simple assignment, without using `with`?

Reply:
I think that'd change the behavior - it'd fail the match instead of returning the unmatched value directly...

Reply:
Ah, I didn't realize that `with` would do that, or that the unmatched value should be returned. Cool!